src/Services/privateApiService.js

```diff
@@ -14,11 +14,15 @@ const config = {
   },
 };
 
-const Get = () => {
-  axios
-    .get('https://jsonplaceholder.typicode.com/users', config)
-    .then((res) => console.log(res))
-    .catch((err) => console.log(err));
+const Get = async (route, id = null) => {
+  const url = 'http://ongapi.alkemy.org/public/api/';
```

Comment:
the route should be all the url to make this dynamic so please delete this url and use directely the route